src/reducers/web3Reducer.ts

```diff
@@ -15,7 +15,8 @@ const initialState: IWeb3State = {
   networkId: 0, // unknown network
 };
 
-const web3Reducer = (state = initialState, action: any) => {
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
why is this needed? I suspect it is because you didn't figure out a better alternative to any. I think there likely is a better way and would prefer that we find it.